docs/getting-started-guides/mesos.md

```diff
@@ -42,9 +45,9 @@ The internal IP address of the master may be obtained via `hostname -i`.
 
 ```bash
 $ export servicehost=$(hostname -i)
-$ export mesos_master=${servicehost}:5050
 $ export KUBERNETES_MASTER=http://${servicehost}:8888
 ```
+
```

Comment:
comment on 'sudo docker run' line a few lines below: this will almost always work, but the etcd version that we have actually qualified Kubernetes against can be found in "cluster/images/etcd/Makefile" which may not be the latest etcd. Probably not worth mentioning though.